docs/conf/portland/2022/lightning-talks.rst

```diff
@@ -0,0 +1,86 @@
+:template: {{year}}/generic.html
+
+Lightning Talks
```

Comment:
How about throwing another flag on the barbie for hiding this until we sort it out? I hope it's not a big PITA ;)